### PR TITLE
Add audit to CI

### PR DESCRIPTION
**Pull request by ryanwi**
Add audit to CI and fix several vulnerabilities.

Fixes:

* CVE-2021-27515
* CVE-2021-23337
* CVE-2020-7733
* CVE-2021-27292

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -35,7 +35,10 @@ jobs:
       # Runs a single command using the runners shell
       - name: NPM Install
         run: npm install
-      
+
+      - name: NPM Audit
+        run: npm audit --audit-level=(high|critical)
```
Comment (ryanwi) [inline .github/workflows/main.yml:40]:
```suggestion
        run: npm audit --audit-level=high
```

**Comment by ryanwi** [comment]
Maybe start with this? The remaining highs are transitive dependencies that require other package updates.

**Review by bradcypert — APPROVED** [review]

**Review by bradjteamsnap — APPROVED** [review]

**Comment by bradjteamsnap** [comment]
Should we also add some text to the contributing.md about updating the packages if the audit step fails?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -10,6 +10,7 @@ Once your user is created and added to the org this guide details the steps to p
 
 * rebase and get  your branch ahead of master
 * then from your working branch
+* run `npm audit --audit-level=high` to ensure there are no vulnerabilities with level high or critical.
```
Comment (ryanwi) [inline CONTRIBUTING.md:13]: @bradjteamsnap is this sufficient or expand more?

Comment (bradjteamsnap) [reply CONTRIBUTING.md:13]: Works for me :)

**Pull request merged**